iOS_Project/Module/Favourite/FavouriteViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  FavouriteViewModel.swift
+//  iOS_Project
+//
+//  Created by Asalah Sayed on 26/05/2023.
+//
+
+import Foundation
+import CoreData
+class FavouriteViewModel{
```

Comment:
please add a protocol to be confirmed and use dependency injection

Reply:
Resolved